sda-commons-web-autoconfigure/src/main/java/org/sdase/commons/spring/boot/web/security/SdaWebSecurityConfiguration.java

```diff
@@ -8,10 +8,23 @@
 package org.sdase.commons.spring.boot.web.security;
 
 import org.springframework.boot.autoconfigure.AutoConfiguration;
+import org.springframework.boot.web.servlet.FilterRegistrationBean;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ComponentScan;
+import org.springframework.core.Ordered;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.web.filter.ForwardedHeaderFilter;
 
 @EnableWebSecurity
 @ComponentScan
 @AutoConfiguration
-public class SdaWebSecurityConfiguration {}
+public class SdaWebSecurityConfiguration {
+  @Bean
+  public FilterRegistrationBean<ForwardedHeaderFilter> forwardedHeaderFilter() {
+    ForwardedHeaderFilter filter = new ForwardedHeaderFilter();
+    FilterRegistrationBean<ForwardedHeaderFilter> registration =
+        new FilterRegistrationBean<>(filter);
+    registration.setOrder(Ordered.HIGHEST_PRECEDENCE);
+    return registration;
+  }
```

Comment:
Do we want to document the hardening requirement here?

Reply:
I extracted all documentation into the last PR, because it was impossible to separate the changes in the markdown file. I'll check there, if it is mentioned properly.

Reply:
It's covered [here](https://github.com/SDA-SE/sda-spring-boot-commons/blob/43e6dbfec192e2c7a940358ced583318ea6890bf/docs/security/index.md#risk-loss-of-source-ip-address).